src/sentry/static/sentry/app/views/workBatchDetails/shared/workBatchDetails.jsx

```diff
@@ -188,31 +51,22 @@ const WorkBatchDetails = createReactClass({
     } else {
       throw new Error('Unexpected tab id ' + tab.id);
     }
-  },
-
-  render() {
-    const params = this.props.params;
-    const {workBatch} = this.state;
+  }
 
-    if (this.state.error) {
-      switch (this.state.errorType) {
-        case ERROR_TYPES.GROUP_NOT_FOUND:
-          return (
-            <div className="alert alert-block">
-              {t('The issue you were looking for was not found.')}
-            </div>
-          );
-        default:
-          return <LoadingError onRetry={this.remountComponent} />;
+  activeTab() {
```

Comment:
activeTab is not used.

Reply:
Right! WorkBatchDetails is just there as a mock. I'll clean it up in my current task.